src/Adapter/Tax/TaxComputer.php

```diff
@@ -69,14 +69,14 @@ public function computePriceWithTaxes(
     /**
      * @param DecimalNumber $priceTaxIncluded
      * @param TaxRulesGroupId $taxRulesGroupId
-     * @param CountryId $countryId
+     * @param CountryIdInterface $countryId
      *
      * @return DecimalNumber
      */
     public function computePriceWithoutTaxes(
         DecimalNumber $priceTaxIncluded,
         TaxRulesGroupId $taxRulesGroupId,
-        CountryId $countryId
+        CountryIdInterface $countryId
```

Comment:
Just to be sure, does it mean we can get taxes for a countryId equal to zero? Because I'm not sure it's possible, maybe it shouldn't be allowed.

It seems like the perfect example where using `CountryId` ensures that we actually have a valid non zero ID that is provided Meaning it's the calling code that has the responsibility of providing appropriate input, and handling the fallback when it can't

Reply:
Don't resolve conversations if you don't answer them please... it makes reviewing work even more complicated and it hides the questions

Reply:
We need to check the use of this code to see if 0 is a valid value in this case But since we can't get a country(s tax rates without a country's ID I doubt it IF so the we should force the use of `CountryId` here not use the interface